bundles/org.openhab.binding.bluetooth.radoneye/src/main/java/org/openhab/binding/bluetooth/radoneye/internal/AbstractRadoneyeHandler.java

```diff
@@ -45,10 +44,9 @@ abstract public class AbstractRadoneyeHandler extends BeaconBluetoothHandler {
     private final Logger logger = LoggerFactory.getLogger(AbstractRadoneyeHandler.class);
 
     private AtomicInteger sinceLastReadSec = new AtomicInteger();
-    private Optional<RadoneyeConfiguration> configuration = Optional.empty();
+    private @Nullable RadoneyeConfiguration configuration;
```

Comment:
To avoid null check warnings, please define configuration like that:
```
private RadoneyeConfiguration configuration = new RadoneyeConfiguration();
```